<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Density of States</string>
   </attribute>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plotLayoutDos"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Input File</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browse">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+B</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Range</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_start">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>-50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_stop">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_plot">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+P</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Freq. Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intmin">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intmax">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>100000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Freq. Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Int. max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Int. min</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Band Structures</string>
   </attribute>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plotLayoutBand"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Input File information.json</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_band">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browse_band">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+B</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Energy Range</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_band_min">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>-50</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Energy Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Energy Max</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_band_max">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_plot_band">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+P</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_orbital">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>10</y>
      <width>231</width>
      <height>551</height>
     </rect>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Atoms</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_check_orbital">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reduce Orbital</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>Page</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pushButton_browse</tabstop>
  <tabstop>lineEdit_start</tabstop>
  <tabstop>lineEdit_stop</tabstop>
  <tabstop>lineEdit_intmax</tabstop>
  <tabstop>lineEdit_intmin</tabstop>
  <tabstop>pushButton_plot</tabstop>
  <tabstop>plainTextEdit_band</tabstop>
  <tabstop>pushButton_browse_band</tabstop>
  <tabstop>lineEdit_band_max</tabstop>
  <tabstop>lineEdit_band_min</tabstop>
  <tabstop>pushButton_plot_band</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
